### DISCUSSION TITLE
How to push to large_media

### DISCUSSION THREAD

**Discussion by reverendbedford** [idaholab/moose#18387] [Q&A General] [answered]
I have some large media files that belong in the `large_media` submodule as part of PR #17475 .  It's a submodule so don't I have to first update the `large_media` repository and then update my ref to it in `moose`?  If so how can I submit changes to `large_media`?

**Comment by hugary1995** [comment, marked as answer]
Oh sorry I thought I replied to you in that PR but it turns out I never hit that "comment" button...

Anyways, I think you should submit a PR to large_media. @aeslaughter or someone else with write access will merge it. Then you can update the large media commit in your moose PR.

**Reply by reverendbedford** [reply]
Thanks.  I don't see any PRs on github against `large_media` but maybe I'm just the first person in awhile to not have direct write access that wants to add something.

**Reply by hugary1995** [reply]
The repository is here: https://github.com/idaholab/large_media

There have been 16 closed PRs already.

At least I don't have write access to it either 😃

**Reply by reverendbedford** [reply]
Fair enough, I opened the PR.
